azure-pipelines.yml

```diff
@@ -6,6 +6,8 @@ trigger:
   branches:
     include:
     - '*'
+    exclude:
+    - 'release'
```

Comment:
The public pipeline does not have the permission to run `PipAuthenticate@1` task, so skip it. This also saves some resource during CLI release.